ksm/even_odd.js

```diff
@@ -0,0 +1,8 @@
+function solution(num) {
+    let even="Even";
+    let odd="Odd";
+    if (num%2==0) {
```

Comment:
```suggestion
    let even = "Even";
    let odd = "Odd";
    if (num % 2 === 0) {
```
연산자 사이는 띄어쓰기가 기본적인 컨벤션입니다 :)